src/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,347 @@
+  }
+
+  &:only-child {
+    @include u-padding-y($theme-card-padding-perimeter);
+  }
+}
+
+// Footer
+// ---------------------------------
+.usa-card__footer {
+  @include u-padding-bottom($theme-card-padding-perimeter);
+  @include u-padding-top($theme-card-padding-y / 2);
+  @include u-padding-x($theme-card-padding-perimeter);
+}
+
+.usa-card__footer .usa-button:only-of-type {
+  @include u-margin-right(0);
+}
+
+.usa-card__header,
+.usa-card__body {
+  // strip spacing from first and last content
+  > :last-child {
+    @include u-padding-bottom(0);
+    @include u-margin-bottom(0);
+  }
+
+  > :first-child {
+    @include u-margin-top(0);
+    @include u-padding-top(0);
+  }
+
+  > :only-child {
+    @include u-margin-y(0);
+    @include u-padding-y(0);
+  }
+}
+
+// ---------------------------------
+// Variations
+// ---------------------------------
+
+// Exdent
+.usa-card__header--exdent,
+.usa-card__media--exdent,
+.usa-card__footer--exdent {
+  @include u-margin-x(-$theme-card-border-width);
+}
+
+.usa-card__header--exdent,
+.usa-card__footer--exdent {
+  > * {
+    @include u-padding-x($theme-card-border-width);
+  }
+}
+
+.usa-card__media--exdent {
+  @include u-margin-top(-$theme-card-border-width);
+
+  .usa-card__img {
+    @include u-radius-top($theme-card-border-radius);
+  }
+}
+
+// Header first
+// ---------------------------------
+.usa-card--header-first {
+  .usa-card__header {
+    @include card-inner-radius;
+    @include u-padding-bottom($theme-card-padding-y);
+  }
+
+  .usa-card__header--exdent {
+    @include u-margin-top(-$theme-card-border-width);
+    @include u-radius-top($theme-card-border-radius);
+  }
+
+  .usa-card__media--inset {
+    @include u-padding-top(0);
+  }
+
+  .usa-card__media {
+    @include u-order(0);
+  }
+
+  .usa-card__img {
+    @include u-radius(0);
+  }
+
+  .usa-card__body {
+    @include u-padding-top($theme-card-padding-y);
+  }
+}
+
+.usa-card--flag {
+  @include at-media($theme-card-flag-min-width) {
+    .usa-card__media {
+      @include u-display("flex");
+      @include u-overflow("hidden");
+      @include u-pin-y;
+      @include u-pin-left;
+      @include u-position("absolute");
+      @include u-width($theme-card-flag-image-width);
+    }
+    .usa-card__img {
+      @include u-radius(0);
+      border-top-left-radius: calc(
+        #{radius($theme-card-border-radius)} - #{units(
+            $theme-card-border-width
+          )}
+      );
+      border-bottom-left-radius: calc(
+        #{radius($theme-card-border-radius)} - #{units(
+            $theme-card-border-width
+          )}
+      );
+    }
+
+    .usa-card__header,
+    .usa-card__body,
+    .usa-card__footer {
+      @include u-margin-left($theme-card-flag-image-width);
+    }
+
+    .usa-card__media--exdent {
+      @include u-margin-left(-$theme-card-border-width);
+      @include u-margin-right(0);
+      @include u-margin-y(-$theme-card-border-width);
+      margin-right: 0;
```

Comment:
Do we need all these margins? We have right twice.

Reply:
That _does_ look redundant!